_docs/style/text-conventions/voice-and-tone.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Voice and tone
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Use these guidelines to write in-product text that is friendly, helpful, and inspires confidence.
+resource: false
+status: complete
+last-modified: 2018-06-01
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+The primary job of the words in a UI is to help users complete tasks with no
+confusion and minimal interruption. However, the voice and tone that we use
+with words in a UI also influence how people think and feel about Rackspace.
+
+Voice is our style, our point of view, and our personality. Tone is our
```

Comment:
This paragraph is repetitive and a little difficult to follow. maybe streamline it a bit:

**Voice** is our style, and communicates our personality to the user. 
**Tone** is our mood, and communicates our attitude about the subject to the user.